README.md

```diff
@@ -1 +1,43 @@
-# First-Portfolio
\ No newline at end of file
+# First Portfolio Website
+
+# Built with
```

Comment:
- [OPTIONAL] kindly don't forget the importance of the README file. Its biggest importance is the fact that it helps you to explain your code kilometers in words. Be sure to add the description of your project for more understanding.